pkg/identityserver/blacklist/builtin.go

```diff
@@ -15,148 +15,687 @@
+	"products",
 	"profile",
+	"project",
+	"projects",
+	"promo",
+	"pub",
+	"public",
+	"purpose",
+	"put",
+	"python",
+	"query",
+	"random",
+	"ranking",
+	"read",
+	"readme",
+	"recent",
+	"recruit",
+	"recruitment",
+	"redirect",
+	"redirect-uri",
+	"register",
+	"registration",
+	"release",
+	"remove",
+	"replies",
+	"report",
+	"reports",
+	"repositories",
+	"repository",
+	"req",
+	"request",
+	"requests",
+	"reset",
+	"roc",
+	"root",
+	"router",
+	"rss",
+	"ruby",
+	"rule",
+	"sag",
+	"sale",
+	"sales",
+	"sample",
+	"samples",
+	"save",
+	"school",
+	"script",
+	"scripts",
+	"search",
 	"secure",
+	"security",
+	"self",
+	"send",
+	"server",
+	"server-info",
+	"server-status",
+	"service",
+	"services",
+	"session",
+	"sessions",
+	"setting",
+	"settings",
+	"setup",
+	"share",
+	"shop",
+	"show",
+	"sign-in",
+	"sign-up",
+	"signin",
+	"signout",
+	"signup",
+	"site",
+	"sitemap",
+	"sites",
+	"smartphone",
+	"smtp",
+	"soporte",
+	"source",
+	"spec",
+	"special",
+	"sql",
+	"src",
 	"ssh",
+	"ssl",
+	"ssladmin",
+	"ssladministrator",
+	"sslwebmaster",
+	"staff",
+	"stage",
 	"staging",
+	"start",
+	"stat",
+	"state",
 	"static",
 	"statistics",
 	"stats",
 	"status",
+	"store",
+	"stores",
+	"stories",
+	"style",
+	"styleguide",
+	"stylesheet",
+	"stylesheets",
+	"subdomain",
+	"subscribe",
+	"subscriptions",
+	"super",
+	"superuser",
+	"suporte",
+	"support",
+	"svn",
+	"swf",
+	"sys",
+	"sysadmin",
+	"sysadministrator",
+	"system",
+	"tablet",
+	"tablets",
+	"tag",
+	"talk",
+	"task",
+	"tasks",
 	"team",
-	"wiki",
-	"www",
-
-	// TTN
-	"applicationserver",
-	"as",
-	"broker",
-	"cli",
-	"console",
-	"dashboard",
-	"ga",
-	"gatewayagent",
-	"gatewayserver",
-	"gs",
-	"handler",
-	"identityserver",
-	"is",
-	"joinserver",
-	"js",
-	"networkserver",
-	"noc",
-	"ns",
-	"router",
+	"teams",
+	"tech",
+	"telnet",
+	"term",
+	"terms",
+	"terms-of-service",
+	"termsofservice",
+	"test",
+	"test1",
+	"test2",
+	"test3",
+	"teste",
+	"testing",
+	"tests",
+	"theme",
+	"themes",
+	"thethings",
+	"things",
+	"this",
+	"thread",
+	"threads",
+	"tls",
+	"tmp",
+	"todo",
+	"tool",
+	"tools",
+	"top",
+	"topic",
+	"topics",
+	"tos",
+	"tour",
+	"translations",
+	"trends",
+	"tti",
+	"ttn",
+	"ttn-lw-cli",
+	"ttn-lw-stack",
 	"ttnctl",
+	"tutorial",
+	"tux",
+	"tv",
+	"twitter",
+	"undef",
+	"unfollow",
+	"unsubscribe",
+	"update",
+	"upload",
+	"uploads",
+	"url",
+	"usage",
+	"user",
+	"username",
+	"users",
+	"usuario",
+	"vendas",
+	"ver",
+	"version",
+	"video",
+	"videos",
+	"visitor",
+	"watch",
+	"weather",
+	"web",
+	"webhook",
+	"webhooks",
+	"webmail",
+	"webmaster",
+	"website",
+	"websites",
 	"webui",
+	"welcome",
+	"widget",
+	"widgets",
+	"wiki",
+	"win",
+	"windows",
+	"word",
+	"work",
+	"works",
+	"workshop",
+	"ww",
+	"wws",
+	"www",
+	"www1",
+	"www2",
+	"www3",
+	"www4",
+	"www5",
+	"www6",
+	"www7",
+	"wwws",
+	"wwww",
+	"xfn",
+	"xml",
+	"xmpp",
+	"xpg",
+	"xxx",
+	"yaml",
+	"year",
+	"yml",
+	"you",
+	"yourdomain",
```

Comment:
Also add `yourapp`, `yourapplication`, `yourdevice`, `yourorganization`

(for use in examples)

Reply:
Also with `your-`...

And then probably also with `example` prefix instead of `your`